desktop/renderer/remote-tracker.js

```diff
@@ -47,6 +48,7 @@ class RemoteTracker extends Component<void, Props, void> {
             component='tracker'
             username={username}
             startTimer={this.props.startTimer}
+            errorRetry={() => this.props.errorRetry(username)}
```

Comment:
So hacky... 
We need to do this to make sure the side effects run in the main window.

With sagas all action creators are pure and we won't need this.